styles/lichess/catppuccin.user.css

```diff
@@ -68,6 +69,18 @@
     @crust: @catppuccin[@@lookup][@crust];
     @accent-color: @catppuccin[@@lookup][@@accent];
 
+    /* Prettier Lichess Support */
```

Comment:
What is "Prettier Lichess"?

Reply:
Extension that makes lichess look pretty (and changes the color)